openmdao/drivers/tests/test_pyoptsparse_driver.py

```diff
@@ -2270,6 +2270,413 @@ def test_singular_jac_tol(self):
 
         prob.run_driver()
 
+    def test_constraint_alias(self):
```

Comment:
Do we have tests to check that we'll catch it if a user chooses an alias name that collides with a real variable name or with another alias?  I may have missed them.

Reply:
It turns out that it is a bit dangerous to allow them to use existing openmdao names, so I've added the test and the check.